src/components/Landing/index.tsx

```diff
@@ -216,6 +218,30 @@ export default function Landing() {
         </div>
       </section>
 
+      {/* <section
```

Comment:
**question** should we remove this for now while we finalize design? Can always come back to this commit to retrieve the code 😊